index.html

```diff
@@ -43,6 +43,7 @@
         <li><a href="/user/build-timeouts">My builds are timing out, what can I do?</a></li>
         <li><a href="/user/repository-providers">What repository providers can I use?</a></li>
         <li><a href="/user/pricing/">Why can't I find pricing information?</a></li>
+        <li><a href="/user/ci-environment-trusty-beta/">Can I use Ubuntu Trusty for my builds?"</a><li>
```

Comment:
I get this line of reasoning, but eventually Trusty is going to be the default for full VM and containerized builds, with Precise also available, so I am not sure calling it out as it's own thing makes the most sense.

Maybe it does for now and we can rework things more once we're closer to Trusty as the default for both full VM and containerized builds?